.github/workflows/mac_binary.yml

```diff
@@ -3,6 +3,8 @@ on:
   push:
     branches:
       - master
+    tags:
+      - v*
```

Comment:
oh and just to be clear, I didn't mean to suggest changing it to `v*` since that was more an example of showcasing that github rules could be different. It wouldn't work for our releases since we don't tag with `v` at the start